applications/osmo4_ios/osmo4ios-Info.plist

```diff
@@ -27,11 +27,11 @@
 		</dict>
 	</dict>
 	<key>CFBundleIdentifier</key>
-	<string>com.telecom-paristech.gpac</string>
+	<string>net.motiontree.motiontreerdplayer</string>
```

Comment:
I am sure that only one line from my pull request should be used and it is not the one you point to.